docs/tuning.md

```diff
@@ -50,7 +50,7 @@ in your operations) and performance. It provides two serialization libraries:
   Java serialization is flexible but often quite slow, and leads to large
   serialized formats for many classes.
 * [Kryo serialization](https://github.com/EsotericSoftware/kryo): Spark can also use
-  the Kryo library (version 4) to serialize objects more quickly. Kryo is significantly
+  the Kryo library (version 5) to serialize objects more quickly. Kryo is significantly
```

Comment:
This is fine; we can also just delete the version reference here.

Reply:
+1